content/sensu-go/5.16/installation/install-sensu.md

```diff
@@ -77,7 +77,19 @@ sudo yum install sensu-go-backend
 
 {{< /language-toggle >}}
 
-##### 2. Configure and start
+##### 2. Initialize
+
+Run `sensu-backend init` to set up your Sensu administrator username and password. In this initialization step, you only need to provide a username and password string &mdash; no need for role-based access control (RBAC) or etcd.
```

Comment:
> no need for role-based access control (RBAC) or etcd.

The tool connects to etcd, using the configuration supplied in backend.yml, so it's probably best not to mention etcd in this context